programmers/난이도별/level01.K번째수/rockmiin.py

```diff
@@ -0,0 +1,9 @@
+
+def solution(array, commands):
+
+    result=[]
+    for i in range (len(commands)):
+        cut=array[commands[i][0]-1:commands[i][1]]
+        cut=sorted(cut)
+        result.append(cut[commands[i][2]-1])
```

Comment:
```suggestion
    for i, j, k in range commands:
        cut = sorted(array[i-1:j])
        result.append(cut[k - 1])
```

이렇게 더 간결하게 표현할 수 있어요!